awesome_dashboard/static/src/dashboard/piechartcard/pirchart_card.js

```diff
@@ -0,0 +1,16 @@
+import { Component } from "@odoo/owl";
+import { PieChart } from "../pie_chart/pie_chart";
+
+export class PieChartCard extends Component {
+    static components = { PieChart }
+    static template = "awesome_dashboard.piechartcard";
```

Comment:
```suggestion
    static template = "awesome_dashboard.PieChartCard";
```
name templates in PascalCase